common/changes/@microsoft/rush/ianc-rush-link-absolute_2018-11-09-01-48.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "comment": "Include an env variable option to create symlinks with absolute paths.",
```

Comment:
>env [](start = 29, length = 3)

"env" is not a word

Reply:
Say the name of the variable

---
In reply to: [232122208](https://github.com/Microsoft/web-build-tools/pull/942#discussion_r232122208) [](ancestors = 232122208)